app/views/templates/root_node.html.haml

```diff
@@ -29,12 +29,14 @@
 
 
   %h2.home-heading News
-
+  - if NewsArticle.published.all.empty?
```

Comment:
There was a conversation about that on Slack. I couldn't really decide where any of this should go.